homeassistant/components/omie/coordinator.py

```diff
@@ -0,0 +1,74 @@
+"""Coordinator for the OMIE - Spain and Portugal electricity prices integration."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+import datetime as dt
```

Comment:
if possible, lets use `now` from `homeassistant.util.dt` instead of `datetime.datetime`